root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js

```diff
@@ -143,22 +151,19 @@ function Fx_OnSubmit() {
     // ・ASP.NET Ajax Extension
     // のどちらも、通過する。
 
-    // ・IE6.0のhrefのdoPostBackだと動かない。
-    // ・OperaのhrefのdoPostBack（二重送信時限定）だと動かない。
-
-    // alert(navigator.appName);
-    // alert(navigator.appVersion);
-
     // ----------
 
+    // In Chrome, Safari and Firefox, document.readyState always returns complete
+    // Hence double transmission is prevented using Form_IsSubmitted flag
+
     // ↑Operaの場合、ポスト後直ちにタイマーが無効になるので、実質機能しない。
     //   ※ ただし、Ajaxの場合は、タイマーが無効にならない。
```

Comment:
@yusukemaegawa 

This is comment wrong ?
I deleted comment of above.

##### see :
https://github.com/SymphonyTeleca/OpenTouryo/blob/nishino/root/programs/C%23/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js#L166